src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreSummaryRequest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using NuGet.Configuration;
+using NuGet.Logging;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.Commands
+{
+    /// <summary>
+    /// Wrapper for RestoreRequest
+    /// </summary>
+    public class RestoreSummaryRequest
```

Comment:
Could this be collapsed down into `RestoreRequest`? It seems like each `RestoreSummaryRequest` has one `RestoreRequest` (1 -> 1 relationship)?

If not, this needs better docs. I am not sure what it represents or how it is different than `RequestRequest`.